build/husky.sh

```diff
@@ -0,0 +1,4 @@
+echo "Husky installing ..."
+npm install husky --save-dev
```

Comment:
We do not want to add husky as a dependency each time the `postinstall` script is executed. This command should be done one time so that husky is added as a dev dependency which would be reflected in this PR's `package.json` changes.